config/sync/user.mail.yml

```diff
@@ -6,7 +6,7 @@ cancel_confirm:
   body: "[user:display-name],\r\n\r\nA request to cancel your account has been made at [site:name].\r\n\r\nYou may now cancel your account on [site:url-brief] by clicking this link or copying and pasting it into your browser:\r\n\r\n[user:cancel-url]\r\n\r\nNOTE: The cancellation of your account is not reversible.\r\n\r\nThis link expires in one day and nothing will happen if it is not used.\r\n\r\n--  [site:name] team"
 password_reset:
   subject: 'Replacement login information for [user:display-name] at [site:name]'
-  body: "[user:display-name],\r\n\r\nA request to reset the password for your account has been made at [site:name].\r\n\r\nYou may now log in by clicking this link or copying and pasting it into your browser:\r\n\r\n[user:one-time-login-url]\r\n\r\nThis link can only be used once to log in and will lead you to a page where you can set your password. It expires after one day and nothing will happen if it's not used.\r\n\r\n--  [site:name] team"
+  body: "[user:display-name],\r\n\r\nA request to reset the password for your account has been made at [site:name].\r\n\r\nYou may now log in by clicking this link or copying and pasting it into your browser:\r\n\r\n[user:one-time-login-url]\r\n\r\nThis link can only be used once to log in and will lead you to a page where you can set your password. It expires after one day and nothing will happen if it's not used.\r\n\r\n[site:name] team"
```

Comment:
Did you mean to change this? It's the same as the other two bodies

Reply:
Yes, the removal of the double hyphen is deliberate. This is because it renders strangely:

<img width="692" alt="image" src="https://github.com/ministryofjustice/prisoner-content-hub-backend/assets/440637/70294c92-0bdc-453e-8f3c-22f73c69c204">

vs when the double hyphen is removed

<img width="687" alt="image" src="https://github.com/ministryofjustice/prisoner-content-hub-backend/assets/440637/8989d573-81e7-4a35-86cc-2e2e2d92c213">